app/controllers/api/base_controller.rb

```diff
@@ -26,23 +26,9 @@ class BaseController < ApplicationController
     include CompressedIds
     extend ErrorHandler::ClassMethods
 
-    #
-    # To skip CSRF token verification as API clients would
-    # not have these. They would instead dealing with the /api/auth
-    # mechanism.
-    #
-    if Vmdb::Application.config.action_controller.allow_forgery_protection
-      skip_before_action :verify_authenticity_token,
-                         :only => [:show, :update, :destroy, :handle_options_request, :options]
-    end
```

Comment:
@abellotti this, AFAIK, is undoing the `protect_from_forgery` declaration from `ApplicationController`, which is made under the same condition. So since it's not there anymore, we don't need to skip this hook, because it's no longer registered.

Worth noting though that it did (as I'm sure you have seen) trigger a brakeman warning. I guess it is not that smart, because technically nothing has changed vis-a-vis `protect_from_forgery`. I should stress though that I don't want to rush this PR - I'd appreciate going through it with a fine-toothed comb together if time allows :musical_note: